examples/some-issuer/frontend/README.md

```diff
@@ -0,0 +1,16 @@
+# SoMe Web3 ID Issuer dApp
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn build-[telegram|discord]`
+
+Builds the app for production to the `dist/[telegram|discord]` folder.
+
+### `yarn dev-[telegram|discord]`
+
+Runs the frontend for the corresponding issuer in the development mode.
```

Comment:
We should explain here that you need to configure some things for this to actually work.

It'd be good to list the environment variables that enable this.

Reply:
Yes, good point. Will add this